packages/manager/apps/web/client/app/css/less/common/global.less

```diff
@@ -166,3 +166,12 @@ code {
     white-space: normal;
   }
 }
+
+// Add a beta status behind a label
+.status-beta::after {
```

Comment:
Looks like a component we could use outside of this project ! :)

Also, not sure we write `Beta` in every languages, did you check ? For example, in French you are supposed to write `Bêta` 😄